lib/matplotlib/mlab.py

```diff
@@ -3188,9 +3188,7 @@ def get_justify(colname, column, precision):
         ntype = column.dtype
 
         if np.issubdtype(ntype, str) or np.issubdtype(ntype, bytes):
-            # The division below handles unicode stored in array, which could
-            # have 4 bytes per char
-            length = max(len(colname), column.itemsize // column[0].itemsize)
+            length = max(len(colname), len(column[0]))
```

Comment:
`len()` is not the right thing.  The best I can come up with is `int(a.dtype['s'].str[2:])`